### DISCUSSION TITLE
想用electron跨端的客户端，问问token的写法怎么实现的

### DISCUSSION THREAD

**Discussion by Admin6016** [Coolapk-UWP/Coolapk-Lite#62] [Q&A]
rt，想用electron跨端的客户端，问问token的写法怎么实现的？我试了github的几个项目都没成功，请教下有test demo吗？

**Comment by wherewhere** [comment]
你翻译一下就行了
https://github.com/Coolapk-UWP/Coolapk-Lite/blob/master/CoolapkLite%2FCoolapkLite%2FCommon%2FTokenCreator.cs
